pkgs/applications/misc/synergy/default.nix

```diff
@@ -5,41 +5,21 @@
 
 stdenv.mkDerivation rec {
   pname = "synergy";
-  version = "1.11.1";
+  version = "1.13.1.41";
 
   src = fetchFromGitHub {
     owner = "symless";
     repo = "synergy-core";
     rev = "${version}-stable";
-    sha256 = "1jk60xw4h6s5crha89wk4y8rrf1f3bixgh5mzh3cq3xyrkba41gh";
+    fetchSubmodules = true;
```

Comment:
Can we just add googleTest to checkInputs instead?

Reply:
No, that won't work, since the build script checks for the submodule and fails if it doesn't exist. Also, it builds everything in the same target - the check just runs one of the binaries built along with everything else.